snippets/cpp/VS_Snippets_Remoting/Httpwebrequest_HaveResponse/CPP/httpwebrequest_haveresponse.cpp

```diff
@@ -23,15 +23,15 @@ int main()
 {
    try
    {
-      
```

Comment:
Many whitespace changes, are they intentional?

Reply:
Not really, but I'm getting lazy reverting them and they don't hurt. I have my editor set to trim them since they are criminal offense in runtime repo :smile: so that's how they appeared here.